updating/preparing-eus-eus-upgrade.adoc

```diff
@@ -7,20 +7,26 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 Due to fundamental Kubernetes design, all {product-title} updates between minor versions must be serialized.
-You must update from {product-title} 4.10 to 4.11 and then to 4.12. You cannot update from {product-title} 4.10 to 4.12 directly.
-However, beginning with the update from {product-title} 4.10 to 4.11 to 4.12, administrators who wish to update between two Extended Update Support (EUS) versions can do so incurring only a single reboot of non-control plane hosts.
+You must update from {product-title} 4.y to 4.y+1, and then to 4.y+2. You cannot update from {product-title} 4.y to 4.y+2 directly.
+However, administrators who wish to update between two Extended Update Support (EUS) versions can do so incurring only a single reboot of non-control plane hosts.
+
+[IMPORTANT]
+====
+EUS-to-EUS updates are only viable on *even-numbered versions* of {product-title}.
```

Comment:
maybe "between" instead of "on"?  Because an EUS-to-EUS update will always touch on a 4.odd midpoint.